### PR TITLE
feat: add chatlayer organization

### PR DESCRIPTION
**Pull request by krvajal**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: src/website/data/organisations.yml [hunk 1]
```diff
@@ -189,3 +189,7 @@
 - name: "Yeovil District Hospital NHS Foundation Trust"
   image: "ydhnhsft.svg"
   link: "https://yeovilhospital.co.uk"
+
+- name: "Chatlayer by Sinch"
```
Comment (luisorbaiceta) [inline src/website/data/organisations.yml:193]: Could you place this in alphabetical order?

Comment (climba03003) [inline src/website/data/organisations.yml]: Please follow the alphabetical order

**Review by climba03003 — CHANGES_REQUESTED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

File: src/website/data/organisations.yml (hunk 1 above)
Comment (krvajal) [reply src/website/data/organisations.yml]: I wasn't aware it was to be alphabetically and I didn't want our logo to show up before other logos that were already added. That's why I put it at the end. But more than happy to put it up in the list ;)

Comment (krvajal) [reply src/website/data/organisations.yml]: Fixed!

**Review by climba03003 — APPROVED** [review]
LGTM.

**Pull request merged**